opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -857,13 +857,10 @@ export const elFindByIds = async (context, user, ids, opts = {}) => {
     if (isStixObjectAliased(type) || forceAliases) {
       elementTypes.push(INTERNAL_IDS_ALIASES);
     }
-    for (let i = 0; i < workingIds.length; i += 1) {
-      const id = workingIds[i];
-      for (let indexType = 0; indexType < elementTypes.length; indexType += 1) {
-        const elementType = elementTypes[indexType];
-        const term = { [`${elementType}.keyword`]: id };
-        idsTermsPerType.push({ term });
-      }
+    for (let indexType = 0; indexType < elementTypes.length; indexType += 1) {
+      const elementType = elementTypes[indexType];
+      const terms = { [`${elementType}.keyword`]: workingIds };
+      idsTermsPerType.push({ terms });
```

Comment:
Why did you rework that part ?

Reply:
yes it's a good question, I had to use this method to resolve entities by ids (ids that I get from the aggregation query) and I saw that we used "term" query with multiple values instead of a "terms" query.